.github/workflows/wan-perf.yml

```diff
@@ -92,11 +95,13 @@ jobs:
         name: bin
         path: artifacts/bin
     - name: Run WAN Perf (QUIC only)
-      if: ${{ github.event_name == 'pull_request' }}
+      #if: ${{ github.event_name == 'pull_request' }}
+      if: ${{ github.event_name == 'workflow_dispatch' }}
```

Comment:
Instead of removing pull request, make it pull request OR workflow_dispatch (similar comment for below).